ops/dev/main.tf

```diff
@@ -38,6 +38,16 @@ resource "azurerm_storage_queue" "test_event_exceptions_queue" {
   storage_account_name = azurerm_storage_account.app.name
 }
 
+resource "azurerm_storage_queue" "flu_test_event_queue" {
+  name                 = "flu-test-event-publishing"
+  storage_account_name = azurerm_storage_account.app.name
+}
+
+resource "azurerm_storage_queue" "flu_test_event_exceptions_queue" {
+  name                 = "flu-test-event-publishing-exceptions"
+  storage_account_name = azurerm_storage_account.app.name
+}
+
```

Comment:
@rin-skylight / @alismx I see the terraform plan looks OK to me. If I wanted to test this out and see the queue terraform creates, could I just deploy this to a dev environment and check in azure?

Reply:
Yes, this is the recommended approach! If you can conduct a test, that would be greatly appreciated. I would like to make sure that things are successful before we attempt to run this against `prod`.